sqld/src/replication/replica/injector.rs

```diff
@@ -20,6 +21,7 @@ impl<'a> FrameInjector<'a> {
                 | OpenFlags::SQLITE_OPEN_NO_MUTEX,
             &INJECTOR_METHODS,
             hook_ctx,
+            DEFAULT_AUTO_CHECKPOINT,
```

Comment:
Please add a comment why it's ok to keep using default auto checkpoint for the replication. It's not obvious.